.github/workflows/publishbot.yml

```diff
@@ -30,12 +23,7 @@ jobs:
   deploy:
     name: 'Deploy the package'
     needs: npm-publish
-    runs-on: [self-hosted, linux, x64, ec2-alpha]
-    container:
```

Comment:
It looks like `node` had trouble installing in the container, so I removed it. It's not necessary when installing the deployment package from `pip`.